build.gradle

```diff
@@ -114,7 +123,7 @@ subprojects {
 
     // Don't publish examples subproject
     if (it.name != 'examples') {
-        def isAndroid = it.name == 'crypto-android'
+        def isAndroid = (it.name == 'crypto-android' || it.name == 'android-utils')
```

Comment:
I believe we can move isAndroid inside Android modules and do not use module names anymore.